src/zlib_h.jl

```diff
@@ -1,11 +1,14 @@
 # general zlib constants, definitions
 
-@unix_only    const _zlib = "libz"
-@windows_only const _zlib = "zlib1"
+@static if is_unix()
+    const _zlib = "libz"
+elseif iswindows()
```

Comment:
It's in Compat: https://github.com/JuliaLang/Compat.jl/blob/546eccbeac2ffe3f80ac4854b7d02e853ead2565/src/Compat.jl#L1167